### PR TITLE
Fix unnecessary scroll in Vulnerability inventory

### LINKED ISSUE #6324: Remove scroll from vulnerabilities inventory table
| Wazuh | Rev  | Browser                      |
|-------|------|------------------------------|
| 4.8.0 | 02 | Chrome, Firefox, Safari, etc |

**Description**
Sometimes the Vulnerabilities Inventory table adds a scroll that has no use. This happens when we search for a value in the search bar that returns just 1 value in the table and then we change the search to return some other results.

**Preconditions**
1. Have alerts in the vulnerabilities dashboard

**Steps to reproduce**
1. Navigate to Vulnerabilities / Inventory
2. Click on the search bar input
3. Search for a value that returns just 1 result
4. Search for another value that returns a few results

**Expected Result**
1. It shouldn't render a scroll in the table

**Actual Result**
1. Renders a scrollbar

https://github.com/wazuh/wazuh-dashboard-plugins/assets/63758389/0f8daa88-2bda-4a79-b491-037ececa3f63

**Screenshots**

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
Fixes the unnecessary scrolling that occurs when you search for something that results in 1 page and then return to a multi-page result.
 
### Issues Resolved
#6324 

### Evidence

https://github.com/wazuh/wazuh-dashboard-plugins/assets/63758389/776cd9c6-8d09-4949-a100-3630f470fad2

### Test
1. Navigate to 'Vunerability/Inventory'
2. Filter table to have 1 page of results
3. Remove filter to have more than 1 page of results
4. Should not be able to scroll the page

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by asteriscos — DISMISSED** [review]
**CR**: :heavy_check_mark: 
**Firefox**: :green_circle: 
Peek 2024-01-19 11-56


**Chrome**: :green_circle: 
Peek 2024-01-19 11-54

File: plugins/main/public/components/overview/vulnerabilities/dashboards/inventory/inventory.scss [hunk 1]
```diff
@@ -1,14 +1,17 @@
 .vulsInventoryContainer {
-    height: calc(100vh - 104px);
-}
-
+  height: calc(100vh - 104px);
 
-.headerIsExpanded .vulsInventoryContainer {
-    height: calc(100vh - 153px);
-}
+  .euiDataGrid__virtualized {
```
Comment (asteriscos) [inline plugins/main/public/components/overview/vulnerabilities/dashboards/inventory/inventory.scss:4]: I suggest adding a comment to make clear why this style is necessary.

Comment (yenienserrano) [reply plugins/main/public/components/overview/vulnerabilities/dashboards/inventory/inventory.scss:5]: done

**Review by asteriscos — APPROVED** [review]

**Review by jbiset — APPROVED** [review]
## CR :green_circle: 

## Test :green_circle: 

### Chrome

[Evidence_Chrome.webm](https://github.com/wazuh/wazuh-dashboard-plugins/assets/43619595/d2b867bf-96fe-4204-98d6-b21fba0ba5f2)

### Firefox

[Evidence_Firefox.webm](https://github.com/wazuh/wazuh-dashboard-plugins/assets/43619595/77f8f890-345c-485e-a82d-776657a6f689)

**Review by asteriscos — APPROVED** [review]

**Review by asteriscos — APPROVED** [review]

**Pull request merged**